sass/variables.scss

```diff
@@ -39,7 +38,7 @@ $side-padding-mob: 10px;
 /* Mixins */
 
 @mixin codetext {
-  font-family: Hack, "Lucida Console", Monaco, "Courier New", monospace;
```

Comment:
not an actual change as we've not been making the webfont available for a very long time